.github/workflows/start_release.yml

```diff
@@ -1,4 +1,4 @@
-name: Publish Release
+name: Start Release
```

Comment:
Nit: can we rename this to make it clearer that this doesn't actually release anything? For example `Draft/Prepare/Stage release`